### PR TITLE
fix the length of sa name is over 63

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

**Comment by xuezhaojun** [comment]
LGTM left for @elgnay to review.

File: pkg/controller/importconfig/cluster_info.go [hunk 1]
```diff
@@ -348,3 +352,11 @@ func getImage(customRegistry, envName string) (string, error) {
 
 	return customImage, nil
 }
+
+func getBootstrapSAName(clusterName string) string {
+	bootstrapSAName := fmt.Sprintf("%s-%s", clusterName, bootstrapSASuffix)
+	if len(bootstrapSAName) > 63 {
+		return fmt.Sprintf("%s-%s", clusterName[:63-len("-"+bootstrapSASuffix)], bootstrapSASuffix)
```
Comment (zhujian7) [inline pkg/controller/importconfig/cluster_info.go:359]:
```suggestion
		return bootstrapSAName[:63]
```

**Pull request merged**